PyPDF2/_merger.py

```diff
@@ -54,9 +54,11 @@
 from .constants import PagesAttributes as PA
 from .constants import TypArguments, TypFitArguments
 from .generic import (
+    DEFAULT_FIT,
```

Comment:
Instead of DEFAULT_FIT, I would say PAGE_FIT

Reply:
Done!